### PR TITLE
Update IBUS telemetry

### PR DESCRIPTION
**Pull request by hali9**
Based on cleanflight doc.

### CODE REVIEW

**Comment by DzikuVx** [comment]
@hali9 small update required

File: src/main/fc/cli.c [hunk 1]
```diff
@@ -846,6 +846,7 @@ static const clivalue_t valueTable[] = {
 #ifdef TELEMETRY_SMARTPORT
     { "smartport_uart_unidir",      VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_TELEMETRY_CONFIG, offsetof(telemetryConfig_t, smartportUartUnidirectional) },
 #endif
+    { "ibus_telemetry_type ",       VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0,  255 }, PG_TELEMETRY_CONFIG, offsetof(telemetryConfig_t, ibusTelemetryType ) },
```
Comment (digitalentity) [inline src/main/fc/cli.c]: Cli.md update is required for this parameter.

File: docs/Cli.md [hunk 2]
```diff
@@ -205,6 +205,7 @@ Re-apply any new defaults as desired.
 |  frsky_vfas_cell_voltage  | OFF |  |
 |  hott_alarm_sound_interval  | 5 | Battery alarm delay in seconds for Hott telemetry |
 |  smartport_uart_unidir  | OFF | Turn UART into UNIDIR for smartport telemetry for usage on F1 and F4 target. See Telemertry.md for details |
+|  ibus_telemetry_type  | 0 | Type compatibility ibus telemetry for transmitters. See Telemertry.md label IBUS for details. |
```
Comment (digitalentity) [inline docs/Cli.md:208]: Don't see anything regarding this setting in `Telemetry.md`.

**Comment by digitalentity** [comment]
@hali9 what's the status of this PR?

**Comment by hali9** [comment]
Ready
Tested on SPRACINGF3

**Pull request merged**